src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,34 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int bucketCapacity;
+    private int boomReach;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. They should be accessed through special methods (getters and setters).